app/renderer/components/Inspector/GestureEditor.js

```diff
@@ -0,0 +1,565 @@
+                bordered={false}
+                maxLength={10}
+                onChange={(e) => {pointer.name = e.target.value; this.setState({pointers});}}/>
+            </Tooltip>}
+            key={pointer.id}>
+            <Row gutter={[24, 24]}>
+              {(pointer.ticks).map((tick) =>
+                <Col xs={12} sm={12} md={12} lg={8} xl={6} xxl={4} key={`${tick.id}.col1`}>
+                  <div>
+                    {tickCard(tick)}
+                  </div>
+                </Col>
+              )}
+              <Col xs={12} sm={12} md={12} lg={8} xl={6} xxl={4} key={`${pointer.id}.col1`}>
+                <Card className={InspectorCSS['tick-plus-card']} bordered={false}>
+                  <center>
+                    <Button className={InspectorCSS['tick-plus-btn']} icon={<PlusCircleOutlined/>}
+                      onClick={() => this.tickAction(pointer.id, null, ACTION_TYPES.ADD)} key={ACTION_TYPES.ADD}/>
+                  </center>
+                </Card>
+              </Col>
+            </Row>
+          </TabPane>
+        ))}
+      </Tabs>;
+
+
+    const timeline = this.updateGestureForTimeline().map((pointer) =>
+      <center key={pointer.id}>
+        <Steps key={pointer.id} className={InspectorCSS['gesture-header-timeline']}
+          style={{'--timelineColor': pointer.color}}>
+          {(pointer.ticks).map((tick) => {
+            if (tick.type !== STATUS.FILLER) {
+              const {type, duration, button, x, y} = tick;
+              const iconStyle = {color: pointer.color};
+              return <Step key='step1' status={tick.customStep || STATUS.FINISH} icon={
+                <Popover placement='bottom'
+                  title={<center key={tick.id}>{DISPLAY[type]}</center>}
+                  content={
+                    <div className={InspectorCSS['timeline-tick-title']}>
+                      {duration !== undefined && <p>Duration: {duration}ms</p>}
+                      {button !== undefined && <p>Button: {button === BUTTONS.LEFT ? 'Left' : 'Right'}</p>}
+                      {x !== undefined && <p>X: {x}{coordType === PIXELS ? 'px' : '%'}</p>}
+                      {y !== undefined && <p>Y: {y}{coordType === PIXELS ? 'px' : '%'}</p>}
+                    </div>
+                  }>
+                  {type === POINTER_MOVE && <RightCircleOutlined key={tick.id} className={InspectorCSS['gesture-header-icon']} style={iconStyle}/>}
+                  {type === POINTER_DOWN && <DownCircleOutlined key={tick.id} className={InspectorCSS['gesture-header-icon']} style={iconStyle}/>}
+                  {type === POINTER_UP && <UpCircleOutlined key={tick.id} className={InspectorCSS['gesture-header-icon']} style={iconStyle}/>}
+                  {type === PAUSE && <PauseCircleOutlined key={tick.id} className={InspectorCSS['gesture-header-icon']} style={iconStyle}/>}
+                </Popover>}/>;
+            } else {
+              return <Step key='step2' status={STATUS.WAIT}
```

Comment:
what does `step2` mean?

Reply:
`step2` are just place holder values within the timeline to keep it the same relative lengths as the other pointer timelines. I made the name more descriptive